data-spring-jdbc/src/main/java/org/stellar/anchor/server/data/JpaTestApplication.java

```diff
@@ -0,0 +1,11 @@
+package org.stellar.anchor.server.data;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class JpaTestApplication {
+  public static void main(String[] args) {
+    SpringApplication.run(JpaTestApplication.class, args);
+  }
+}
```

Comment:
What is this being used for? Is it worth adding a Readme or a comment with instructions?

Reply:
Good point. I have added the reason as the class Javadoc.
```
/**
 * This application is solely for the purpose of Spring Boot Test used in JUnit5. Spring Tests
 * requires the Spring Boot Application to be included in the `main` folder instead of the `test`
 * folder.
 */
```